docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -516,6 +516,29 @@ The Iceberg JDBC catalog is supported for the Iceberg connector.  At a minimum,
 database besides PostgreSQL, a JDBC driver jar file must be placed in the plugin
 directory.
 
+:::{list-table} JDBC catalog configuration properties
+:widths: 40, 60
+:header-rows: 1
+
+* - Property name
+  - Description
+* - `iceberg.jdbc-catalog.driver-class`
+  - JDBC driver class name.
+* - `iceberg.jdbc-catalog.connection-url`
+  - The URI to connect to the JDBC server.
+* - `iceberg.jdbc-catalog.connection-user`
+  - User name for JDBC client.
+* - `iceberg.jdbc-catalog.connection-password`
+  - Password for JDBC client.
+* - `iceberg.jdbc-catalog.catalog-name`
+  - Iceberg JDBC metastore catalog name.
+* - `iceberg.jdbc-catalog.default-warehouse-dir`
+  - The default warehouse directory to use for JDBC.
+* - `iceberg.jdbc-catalog.schema-version`
+  - JDBC catalog schema version.
+    Valid values are `V1` or `V2`. Default value is `V1`.
```

Comment:
It should be "`V0` or `V1`". 
https://github.com/trinodb/trino/blob/dd3d89536394ed3fc137942c90960c24d97b5588/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/jdbc/IcebergJdbcCatalogConfig.java#L28

Reply:
```suggestion
    Valid values are `V1` or `V2`. Defaults to `V1`.
```